cfme/tests/cloud_infra_common/test_discovery.py

```diff
@@ -98,3 +100,50 @@ def _cleanup():
     vm_crud.cleanup_on_provider()
     if_scvmm_refresh_provider(provider)
     wait_for_vm_state_changes(vm_crud)
+
+
+def provider_classes(appliance):
+    required_providers = all_required(appliance.version)
+
+    selected = dict(infra=[], cloud=[], container=[])
+    # we want to collect these provider categories
+    for cat in selected.keys():
+        selected[cat].extend(
+            set(  # quick and dirty uniqueness for types/versions
+                prov.klass
+                for prov in required_providers
+                if prov.category == cat
+            )
+        )
+    return selected
+
+
+@pytest.mark.tier(0)
+def test_provider_type_support(appliance, soft_assert):
+    """Test availability of GCE provider in downstream CFME builds
+
+    BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1671844
```

Comment:
This should be under `Bugzilla`

Reply:
ah thanks, almost forgot I put this here. Team has discussed these since I opened the PR, and we do want to enforce some consistency now. Will update.